streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorContextReverseAdapter.java

```diff
@@ -193,7 +130,7 @@ public Cancellable schedule(final Duration interval, final PunctuationType type,
     @Deprecated
     @Override
     public <K, V> void forward(final K key, final V value, final int childIndex) {
-        delegate.forward(key, value, To.child((currentNode().children()).get(childIndex).name()));
+        deprecatedForwarder.forward(key, value, childIndex);
```

Comment:
Here's where we call the defined behavior instead of delegating to the nonexistent method on the delegate.